ci/buildkite-pipeline.sh

```diff
@@ -256,7 +256,15 @@ EOF
 
   command_step "local-cluster" \
     ". ci/rust-version.sh; ci/docker-run.sh \$\$rust_stable_docker_image ci/test-local-cluster.sh" \
-    50
+    40
+
+  command_step "local-cluster-flakey" \
+    ". ci/rust-version.sh; ci/docker-run.sh \$\$rust_stable_docker_image ci/test-local-cluster-flakey.sh" \
+    10
+
+  command_step "local-cluster-slow" \
+    ". ci/rust-version.sh; ci/docker-run.sh \$\$rust_stable_docker_image ci/test-local-cluster-slow.sh" \
+    25
```

Comment:
This makes me so happy and sad at the same time 😝 
Thank you. Also, TIL `flakey` is a legit spelling

Reply:
Yeah so interesting that there are multiple spellings for that word and not a American/British English thing either apparently